examples/aws/policy_files/permission_boundary_policy.json

```diff
@@ -139,8 +139,8 @@
         "arn:aws:iam:::policy/StreamNative/StreamNativeCloudManagementPolicy",
         "arn:aws:iam:::policy/StreamNative/StreamNativeCloudPermissionBoundary",
         "arn:aws:iam:::policy/StreamNative/StreamNativeCloudRuntimePolicy",
```

Comment:
Don't you need the aws account ID in all these policies?

Reply:
Updated